etc/settings.yaml.in

```diff
@@ -49,17 +90,41 @@ execution:
     final_report:
       threads: 1
       memory: 40G
+    sort_index_bam_bwameth:
+      threads: 12
```

Comment:
I'm not too happy with high numbers for threads.  If you only have 4 cores then 12 threads is going to be a bad idea.

Throughout all pipelines it's not really clear if this is supposed to be used as an input to request cores from a job scheduler (as in: this needs 12 cores, so schedule the job on a node with enough CPUs) or if this is merely a hint.  What happens when I run this on my laptop with 4 cores?  Will it fail?  Do we take the number of threads to be limited by the number of cores of the machine that runs the code?